src/css/styles.css

```diff
@@ -124,3 +140,33 @@ p {
 .has-fw-black {
   font-weight: var(--font-weight-black) !important;
 }
+
+/* General Sizing */
+
+.is-xsm {
+  font-size: var(--t-font-size-xsm) !important;
```

Comment:
you can remove `!important` ... there is no need of that ... are you facing any problem without using it ?

Reply:
No, I'm not facing any issues. I am following the utility pattern in Bootstrap. Which uses utility class to overwrite any existing single style.